src/gracetabnote.js

```diff
@@ -0,0 +1,32 @@
+// [VexFlow](http://vexflow.com) - Copyright (c) Mohit Muthanna 2010.
+// @author Balazs Forian-Szabo
+//
+// ## Description
+//
+// A basic implementation of grace notes
+// to be rendered on a tab stave.
+//
+// See `tests/gracetabnote_tests.js` for usage examples.
+
+import { TabNote } from './tabnote';
+import { Flow } from './tables';
+
+export class GraceTabNote extends TabNote {
+  static get CATEGORY() { return 'gracetabnotes'; }
+  static get SCALE() { return 0.66; }
+
+  constructor(note_struct) {
+    super(note_struct, false);
+    this.render_options.y_shift = 1;
+  }
+
+  getCategory() { return GraceTabNote.CATEGORY; }
+
+  getScale() { return GraceTabNote.SCALE; }
+
+  getFont() { return Flow.TABLATURE_GRACE_FONT; }
```

Comment:
Same thing here. Rather than a static constant, this is probably better as part of `render_options`. So all these getters can remain in `tabnote.js` only, and the only thing you'll need here is the constructor and `CATEGORY`.